devops/modules/2_cluster-configuration/task_3.md

```diff
@@ -57,20 +53,15 @@ In this task, you will configure and deploy a Kubernetes (K8s) cluster on AWS us
    - Terraform code is created or extended to manage AWS resources required for the cluster creation.
    - The code includes the creation of a bastion host.
 
-2. **Cluster Deployment (60 points)**
-
-   - A K8s cluster is deployed using either kOps or k3s.
-   - The deployment method is chosen based on the user's preference and understanding of the trade-offs.
-
-3. **Cluster Verification (10 points)**
+2. **Cluster Verification (50 points)**
 
    - The cluster is verified by running the `kubectl get nodes` command from the local computer.
    - A screenshot of the `kubectl get nodes` command output is provided.
 
-4. **Workload Deployment (10 points)**
+4. **Workload Deployment (30 points)**
 
    - A simple workload is deployed on the cluster using `kubectl apply -f https://k8s.io/examples/pods/simple-pod.yaml`.
-   - The workload runs successfully on the cluster.
+   - A screenshot of the `kubectl get all` command output is provided. (pod named "nginx" should be presented)
```

Comment:
Probably, it's worth to move this to the `submission` part, and here write something like "check that pod named nginx is running in the cluster"?